pkgs/by-name/er/erlang-language-platform/package.nix

```diff
@@ -14,14 +14,14 @@ let
 
   hashes = {
     linux-aarch64-unknown-linux-gnu = "sha256-i6XsOK8csrJ/9TDzltA7mGjdutLZONFiYGV5tqSCy8o=";
-    linux-x86_64-unknown-linux-gnu = "sha256-XK3DPWIdPDoIL10EATa8p1bnlpZaOzOdU0LnuKbj++E=";
+    linux-x86_64-unknown-linux-gnu = "sha256-IZV7IytgGzhgHNIlVuOsC6m6koOhUiP9PIeNsRZbcU0=";
     macos-aarch64-apple-darwin = "sha256-8e5duQYDVFyZejMjuZPuWhg1on3CBku9eBuilG5p1BY=";
     macos-x86_64-apple-darwin = "sha256-dnouUBUUAkMr1h+IJWYamxmk8IC7JdeIUS9/YI0GzOU=";
   };
```

Comment:
I don't think this looks good, though?

Reply:
I've created a PR at #432544 to add an update script (using GitHub releases) addressing this and also align package with the current OTP version. I'd hope someone can take a look.